### PR TITLE
Adjust links in README.md

### LINKED ISSUE #2: Adjust links in README.md
## Acceptance criteria

- [ ] The README.md file is updated with links as specified in the instructions.

## Details

See the README.md file in the repo for details.

### PR DESCRIPTION
**Pull request by tigeryu8900**
In this PR, we change the header, fill out the table, and update the links in README.md.

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,19 @@
...
-  in the text `# STARTER-team01` above
-  to match the name of your repo, i. e., `f23-5pm-2-team01`, then delete
-  this TODO item.
-
-* TODO: Correct the links to repo below, 
-  then delete this TODO.  Replace it with 
-  a link to your repo, e.g. 
-  https://github.com/ucsb-cs156-f23/f23-5pm-2-team01
-
-Repo: https://ucsb-cs156-f23/STARTER-team01
```
Comment (MuMuKow) [inline README.md:28]: Keep the link to the repo, but change it to our repository link

File: README.md [hunk 2]
```diff
@@ -46,15 +27,15 @@ team of up to 5 students can build a proxy similar to the one in the example cod
...
+| `mvn test jacoco:report`                           | Runs JUnit tests, and if all tests pass, computes code coverage.  The code coverage report (Jacoco) can be found in `target/site/jacoco/index.html`      |
+| `mvn package`                                      | Builds the jar file `target/team01-spring-boot-1.0.0.jar`                                                                                                |
+| `mvn spring-boot:run`                              | Runs the code to startup a web server.  Access it via `http://localhost:8080` on the *same machine* where the server is running.  Use CTRL/C to stop it. |
+| `java -jar target/team01-spring-backend-1.0.0.jar` | If done after `mvn package`, this is another way to start up the web server.                                                                             |
+| `mvn test pitest:mutationCoverage`                 | Run [pitest mutation coverage](https://pitest.org).  View `target/pit-reports/index.html` for results (may take a few minutes)                           |
```
Comment (MuMuKow) [inline README.md:38]: This portion shouldn't be modified

**Review by MuMuKow — CHANGES_REQUESTED** [review]
Please address following comments

File: README.md (hunk 1 above)
Comment (tigeryu8900) [reply README.md:28]: The link is actually a line below. I forgot to add the `github.com\` part.

File: README.md (hunk 2 above)
Comment (tigeryu8900) [reply README.md:38]: Oops, that's my IDE reformatting the file

**Review by MuMuKow — APPROVED** [review]
LGTM

**Pull request merged**